Bottom: 9732b293589ea90e000db4ad246935a1b27a7c86
Top:    2aa879ba564942551d3072bfcc737be55dbf1f66
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-05-17 14:47:52 +0100

Refresh of polygon-c-add-option-to

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 753ad24..b2de212 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1558,7 +1558,7 @@ ensure_board_outline (void)
     if (PCB->Data->outline != NULL)
       poly_Free (&PCB->Data->outline);
 
-    PCB->Data->outline = board_outline_poly ();
+    PCB->Data->outline = board_outline_poly (false);
     PCB->Data->outline_valid = true;
   }
 }
